### ISSUE TITLE
Custom env variable not updated in pgsql.conf

### ISSUE THREAD

**Issue by vishantsolanki** [zalando/postgres-operator#1398]
Please, answer some short questions which should help us to understand your problem / question better?

- **Which image of the operator are you using?** e.g. registry.opensource.zalan.do/acid/postgres-operator:v1.6.1
- **Where do you run it - cloud or metal? Kubernetes or OpenShift?** [AWS K8s | GCP ... | Bare Metal K8s]
- **Are you running Postgres Operator in production?** [yes | no]
- **Type of issue?** [Bug report, question, feature request, etc.]

Some general remarks when posting a bug report:
- Please, check the operator, pod (Patroni) and postgresql logs first. When copy-pasting many log lines please do it in a separate GitHub gist together with your Postgres CRD and configuration manifest.
- If you feel this issue might be more related to the [Spilo](https://github.com/zalando/spilo/issues) docker image or [Patroni](https://github.com/zalando/patroni/issues), consider opening issues in the respective repos.

**Comment by vishantsolanki** [comment]
Hi all, can anyone suggest how can we update custom env variable in our master pod pgsql.conf file and as of 
now same pgsql.conf file used by master and slave is there any way to make and pass separate env variable in our pgsql.conf file for master/slave.

**Comment by FxKu** [comment]
No, afaik not easily possible. The config is managed by Patroni and if you change on the pods it is proxied over all members.
